includes/admin/functions.php

```diff
@@ -190,13 +180,19 @@ function amp_editor_core_blocks() {
 }
 
 /**
- * Bootstrap the AMP admin pointer class.
+ * Bootstraps AMP admin classes.
  *
- * @since 1.0
+ * @since 1.5.0
  */
-function amp_admin_pointer() {
+function amp_bootstrap_admin() {
 	$admin_pointers = new AMP_Admin_Pointers();
 	$admin_pointers->init();
+
+	$post_meta_box = new AMP_Post_Meta_Box();
+	$post_meta_box->init();
+
+	$site_health = new SiteHealth();
+	$site_health->init();
 }
```

Comment:
General note: it seems somewhat strange that these instances are completely inaccessible since the variables are scoped in the function. This is probably a wider issue that should be addressed later, as it's worse to pollute the global namespace with more variables. /cc @schlessera

Reply:
Yeah, that's a good point. It's be good to have some access to them.